### ISSUE TITLE
Crash on export of photo with no postal address.

### ISSUE THREAD

**Issue by jslttery** [RhetTbull/osxphotos#172]
I have a photo in my album taken mid-flight and when trying to export with:
> osxphotos export --sidecar xmp --export-as-hardlink --export-by-date --update "./Photos 3 Export"

I get the following crash:

```
Exporting 90150FBE-AFCE-4AF1-B6D4-08B6D38FC311.heic as IMG_0438.HEIC
Traceback (most recent call last):
  File "/usr/local/bin/osxphotos", line 10, in <module>
    sys.exit(cli())
  File "/Library/Python/3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Library/Python/3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Library/Python/3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Library/Python/3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Library/Python/3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Library/Python/3.7/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/Library/Python/3.7/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Library/Python/3.7/site-packages/osxphotos/__main__.py", line 1465, in export
    edited_suffix=edited_suffix,
  File "/Library/Python/3.7/site-packages/osxphotos/__main__.py", line 2053, in export_photo
    dry_run=dry_run,
  File "/Library/Python/3.7/site-packages/osxphotos/photoinfo/_photoinfo_export.py", line 557, in export2
    fileutil,
  File "/Library/Python/3.7/site-packages/osxphotos/photoinfo/_photoinfo_export.py", line 869, in _export_photo
    self.json(),
  File "/Library/Python/3.7/site-packages/osxphotos/photoinfo/photoinfo.py", line 746, in json
    place = self.place.as_dict() if self.place else {}
  File "/Library/Python/3.7/site-packages/osxphotos/placeinfo.py", line 635, in as_dict
    "address": self.address._asdict(),
  File "/Library/Python/3.7/site-packages/osxphotos/placeinfo.py", line 539, in address
    street=addr._street,
AttributeError: 'NoneType' object has no attribute '_street'
```

I've exported the photo manually from Photos with the IPTC information which is attached (added .txt extension to get it to upload).

[IMG_0438.xmp.txt](https://github.com/RhetTbull/osxphotos/files/4809451/IMG_0438.xmp.txt)

I'm on Version 0.29.24 (installed via pip3 on MacOS 10.15.5):
```
> osxphotos --version
osxphotos, version 0.29.24
```

**Comment by RhetTbull** [comment]
I think I've fixed this in [v0.29.26](https://github.com/RhetTbull/osxphotos/releases/tag/v0.29.26) but can't test as I don't have a photo that returns valid reverse-geolocation data but no address as in your example. Try `pip3 install --upgrade osxphotos` and let me know if you still get the problem.

**Comment by jslttery** [comment]
@RhetTbull  -  v0.29.26 process and exports my troublesome photo just fine.

Many thanks for the quick fix and also for an excellent tool. This has resolved a long-standing annoyance I've had with iPhone photos in Photos and everything else in Lightroom. I can now export with hard links from Photos and reference from Lightroom - very very nice.
